docs/v3.x/guides/computed-fields.md

```diff
@@ -0,0 +1,102 @@
+# Computed Fields
+
+In this guide we will see how you can add Computed Fields.
+Computed Fields can be used to return computed or formatted values.
+
+## What are computed fields?
+Computed fields are "virtual" values, dynamically added to response object. They may contain any type of data you provide based on other fields, other models fields or anything you want.
+By definition they are not associated with any database column and cannot be modified directly.
```

Comment:
This isn't entirely true which is why I felt a little confused, the value is being saved into a database column (as you mention this later on in the guide about disabling editing).

Reply:
the code was changed and it is not saved into db anymore

Reply:
@alexandrebodin how it changed exactly?

You said that I can remove computed field (last_name) from model, but in this case we wan't computed field to be visible in strapi admin page. Do you mean that computed field will be visible, even if it was not described in model?

Reply:
@wysher I didn't think you want this in the Admin panel but in the API 🤕  This is a whole other story indeed

Reply:
@alexandrebodin is it ok then? :)